woocommerce_fusion/tasks/sync_sales_orders.py

```
@@ -23,6 +23,26 @@
 )
 
 
+def _get_effective_status_maps(woocommerce_server: str) -> Tuple[Dict, Dict]:
+	"""Use server method to compute effective maps; fallback to base maps if needed."""
+	try:
+		wc_server = frappe.get_cached_doc("WooCommerce Server", woocommerce_server)
+		mapping = wc_server.get_effective_status_mapping()
+		reverse = {v: k for k, v in mapping.items()}
+		return mapping, reverse
```

Ruff (0.13.1)

Consider moving this statement to an `else` block

(TRY300)